src/main/resources/messages.properties

```diff
@@ -55,6 +57,9 @@ public.forgot.password.success.unconnected.account=If you have entered an email
 public.forgot.password.success.unconnected.account.contact=If you have entered an email address that is not connected with an account, you will not receive an email. You will need to 
 public.forgot.password.success.unconnected.account.contact.us.text=contact us
 public.forgot.password.success.unconnected.account.contact.end= to create an account.
+public.user.password.reset.expired.text=For security, your link is only valid for 8 hours.
```

Comment:
Putting the number of hours here seems like something we'll forget to change in the future, but I don't see an easy alternative

Reply:
I agree. I was only following the ticket spec.